test/CapAutomator.t.sol

```diff
@@ -1149,17 +1200,31 @@ contract EventTests is CapAutomatorUnitTestBase {
     }
 
     function test_RemoveSupplyCapConfig() public {
-        vm.prank(owner);
+        vm.startPrank(owner);
+        capAutomator.setSupplyCapConfig(
+            asset,
+            20_000,
+            2_000,
+            24 hours
+        );
         vm.expectEmit(address(capAutomator));
         emit RemoveSupplyCapConfig(asset);
         capAutomator.removeSupplyCapConfig(asset);
+        vm.stopPrank();
     }
 
     function test_RemoveBorrowCapConfig() public {
-        vm.prank(owner);
+        vm.startPrank(owner);
+        capAutomator.setBorrowCapConfig(
+            asset,
+            10_000,
+            1_000,
+            12 hours
+        );
         vm.expectEmit(address(capAutomator));
         emit RemoveBorrowCapConfig(asset);
         capAutomator.removeBorrowCapConfig(asset);
+        vm.stopPrank();
     }
 
     function test_UpdateSupplyCap() public {
```

Comment:
`Update` -> `update`. And check other places this may be incorrectly capitalized on the first word.